examples/index.html

```diff
@@ -219,7 +219,8 @@ <h1><a href="https://threejs.org">three.js</a></h1>
 
 		function updateFilter() {
 
-			var v = filterInput.value;
+			var v = filterInput.value.trim();
+			v = v.replace( /\s+/gi, ' ' ).replace( ' ', '|' );
```

Comment:
What is `v.replace( /\s+/gi, ' ' )` for?
Also, `.replace( ' ', '|' )` only replaces the first space. Is this intentional?

Reply:
> What is `v.replace( /\s+/gi, ' ' )` for?

This replace multiple whitespace to a single whitespace.

> Also, `.replace( ' ', '|' )` only replaces the first space. Is this intentional?

nope it should replace all of them.

Reply:
<img width="291" alt="Screen Shot 2020-06-21 at 4 47 59 PM" src="https://user-images.githubusercontent.com/97088/85237886-14b00d00-b465-11ea-8589-1a72479036e8.png">

Reply:
```js
v = v.replace( /\s+/gi, ' ' ).replace( ' ', '|' );
```
can be written so:
```js
v = v.replace( /\s+/gi, '|' );
```
That should produce the intended result: https://jsfiddle.net/a0L5ef4m/